### PR TITLE
Revert untranslatable fields in meta.yaml

### PR DESCRIPTION
**Pull request by wheleph**

### CODE REVIEW

File: nttt/tidyup.py [hunk 1]
```diff
@@ -71,7 +100,7 @@ def tidyup_translations(arguments):
 
                 print("Fixing - {}".format(file_name))
                 if os.path.basename(source_file_path) == "meta.yml":
-                    fix_meta(source_file_path, output_file_path)
+                    fix_meta(source_file_path, english_folder, output_file_path)
```
Comment (robertjank) [inline nttt/tidyup.py:103]: Function fix_meta expects a file path as second argument (like the other arguments), but here a directory is passed.

File: nttt/tidyup.py [hunk 2]
```diff
@@ -1,11 +1,40 @@
...
+    parsed_md = yaml.safe_load(content)
+    english_parsed_md = yaml.safe_load(english_content)
+
+    translatable_keys = ["title", "description", "steps"]
+    for key in parsed_md:
+        if key not in translatable_keys:
+            parsed_md[key] = english_parsed_md[key]
+
+    class IndentedDumper(yaml.Dumper):
+        def increase_indent(self, flow=False, indentless=False):
```
Comment (robertjank) [inline nttt/tidyup.py:31]: Argument indentless is not used?

File: nttt/tidyup.py [hunk 3]
```diff
@@ -1,11 +1,40 @@
...
+def revert_untranslatable_meta_elements(content, english_content):
+    parsed_md = yaml.safe_load(content)
+    english_parsed_md = yaml.safe_load(english_content)
+
+    translatable_keys = ["title", "description", "steps"]
+    for key in parsed_md:
+        if key not in translatable_keys:
+            parsed_md[key] = english_parsed_md[key]
+
+    class IndentedDumper(yaml.Dumper):
```
Comment (robertjank) [inline nttt/tidyup.py:30]: Maybe add a short desciption why this class is needed.

File: unit_test/test_utilities.py [hunk 4]
```diff
@@ -26,3 +28,43 @@ def __exit__(self, exc_type, exc_val, exc_tb):
...
+            nttt.tidyup.fix_meta(temp_src.name, temp_english_src.name, temp_dest.name)
+
+            result = temp_dest.read().decode('utf-8')
+            unittest.TestCase().assertEqual(result, expected_output_file_content)
+
+    @staticmethod
+    def assert_fix_step(input_file_content, second_argument, expected_output_file_content):
+        """
+        - Prepare input file
+        - Run the given function (that takes as arguments names of input and output files)
```
Comment (robertjank) [inline unit_test/test_utilities.py:59]: This function is always fix_step.

File: nttt/tidyup.py (hunk 2 above)
Comment (robertjank) [inline nttt/tidyup.py:31]: Argument indentless is not used?

File: nttt/tidyup.py (hunk 1 above)
Comment (robertjank) [inline nttt/tidyup.py:103]: Function fix_meta expects a file path as second argument (like the other arguments), but here a directory is passed.

**Review by wheleph — COMMENTED** [review]
@robertjank, I implemented your suggestions. Please have another look

**Pull request merged**